pkgs/top-level/all-packages.nix

```diff
@@ -33473,7 +33473,9 @@ with pkgs;
 
   termpdfpy = python3Packages.callPackage ../applications/misc/termpdf.py {};
 
-  inherit (callPackage ../applications/networking/cluster/terraform { })
+  inherit (callPackage ../applications/networking/cluster/terraform {
+    buildGoModule = buildGo117Module;
```

Comment:
Do you know why 117 is not the default?

Reply:
The `1.17` update is more complex than usual, it has a breaking change with how it handles `vendor` directories.

We need to verify the `vendorSha256` FOD hashes and update them if required.

Rather than trying to push through one large `1.16 -> 1.17` PR which may have hundreds of `vendorSha256` changes I'm planning to do it in smaller batches and then drop the overrides after we've changed the default.